src/Transport/GrpcTransport.php

```diff
@@ -143,8 +153,9 @@ public static function build(string $apiEndpoint, array $config = [])
                 "instead got: " . print_r($channel, true)
             );
         }
+        $logger = isset($config['logger']) ? $config['logger'] : null;
```

Comment:
Use more succinct ternary/null-coalescing:
```suggestion
        $logger = $config['logger'] ?? null;
```
... or do it inline, as I suggest below